hazelcast/src/test/java/com/hazelcast/map/EntryProcessorOffloadableTest.java

```diff
@@ -103,6 +109,7 @@ public Config getConfig() {
 
     @Before
     public void before() {
+        System.setProperty("hazelcast.operation.call.timeout.millis", "4000");
```

Comment:
I would not make use of a System property. You need to take care resetting it and it could cause problems for parallel tests. Best to make use of config.properties.

Reply:
FYI: System properties are saved and restored by our runner implementations. There is also a `OverridePropertyRule` to handle this nicely for a whole test class.

Reply:
:+1: Changed to use the config properties